### DISCUSSION TITLE
Is it really necessary to unconditionally include native dependencies to Avalonia.Skia ?

### DISCUSSION THREAD

**Discussion by gbtb** [AvaloniaUI/Avalonia#16792] [Q&A]
I wonder why Avalonia.Skia has native dependencies for Linux and Webassembly [always included](https://www.nuget.org/packages/Avalonia.Skia/#dependencies-body-tab)? If I target only Linux platform, why do I have to download web assembly stuff as well? I think it either should not depend on native packages at all and require a developer to install necessary packages manually. Or be a bit smarter about runtime identifiers in PackageReferences to only include linux deps for linux, and wasm for wasm.

**Comment by maxkatz6** [comment]
These packages already respect runtime identifier and only copied to the output folder on a correct platform.
Sure, it's always downloaded on the dev's machine during dotnet restore, but that's not really an issue.
